boards/m5stack/m5stack_cores3/Kconfig.m5stack_cores3

```diff
@@ -8,3 +8,13 @@ config BOARD_M5STACK_CORES3
 	select SOC_ESP32S3_PROCPU if BOARD_M5STACK_CORES3_ESP32S3_PROCPU || \
 				     BOARD_M5STACK_CORES3_ESP32S3_PROCPU_SE
 	select SOC_ESP32S3_APPCPU if BOARD_M5STACK_CORES3_ESP32S3_APPCPU
+
+if SOC_ESP32S3_PROCPU
```

Comment:
Please, move this to `Kconfig.defconfig` file:

```c
if BOARD_M5STACK_CORES3_ESP32S3_PROCPU

config INPUT_FT5336_INTERRUPT
	default y if INPUT

config INPUT
	default y

endif # BOARD_M5STACK_CORES3_ESP32S3_PROCPU
```

Reply:
Thank you for your comment.
I applied your suggestion and added modifications to make it compatible with variants.